backend/.env.example

```
@@ -0,0 +1,8 @@
+KEEPER_ENABLED=false
+KEEPER_SECRET_KEY=
+
+ESCROW_CONTRACT_ID=
+STELLAR_RPC_URL=
+STELLAR_NETWORK_PASSPHRASE=Test SDF Network ; September 2015
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STELLAR_NETWORK_PASSPHRASE key should go before the STELLAR_RPC_URL key

(UnorderedKey)